meson.build

```diff
@@ -702,6 +702,7 @@ env = find_program('env')
 rsync = find_program('rsync', required : false)
 diff = find_program('diff')
 find = find_program('find')
+cp = find_program('cp')
```

Comment:
Unused? Please drop this.

Reply:
Yes. unused, some leftover code i forgot to remove.